app/src/org/commcare/appupdate/FlexibleAppUpdateController.java

```diff
@@ -0,0 +1,19 @@
+package org.commcare.appupdate;
+
+import com.google.android.play.core.install.InstallStateUpdatedListener;
+
+/**
+ * Helper for monitoring flexible app updates using playstore AppUpdateManager.
+ * @author $|-|!˅@M
+ */
+public interface FlexibleAppUpdateController extends AppUpdateController, InstallStateUpdatedListener {
```

Comment:
[This comment](https://issuetracker.google.com/issues/133299031#comment3) seems to suggest that it's up to the developer to start an Immediate or Flexible update. 
So only using the Flexible update controller here.